### PR TITLE
Add date breaking change listorders resource

### PR DESCRIPTION
**Pull request by Isabella-Veloso**
#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [x] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

The CX team required a deadline for the product team to start the breaking change, because clients were uncomfortable and unsatisfied with the subtle change.

### CODE REVIEW

File: docs/release-notes/custom-io-apps-need-to-add-the-listorders-resource.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: "Custom IO apps need to add the ListOrders resource"
 createdAt: 2023-02-09T14:51:46.000Z
 hidden: false
 type: "improved"
-excerpt: Accounts with custom IO apps need to add the ListOrders resource on the manifest.json file to be able to call list orders requests.
+excerpt: Starting March 15th, 2023, accounts with custom IO apps will need to have the ListOrders resource on the manifest.json file to be able to call list orders requests.
```
Comment (jumeyermattosarruda) [inline docs/release-notes/custom-io-apps-need-to-add-the-listorders-resource.md:7]:
```suggestion
excerpt: Starting March 15th, 2023, accounts with custom IO apps will need the ListOrders resource on the manifest.json file to be able to call list orders requests.
```

File: docs/release-notes/custom-io-apps-need-to-add-the-listorders-resource.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ The new `ListOrders` resource was automatically added to all authentications ([r
```
Comment (jumeyermattosarruda) [inline docs/release-notes/custom-io-apps-need-to-add-the-listorders-resource.md:16]: O trecho que fala "the new resource was automatically added" talvez fique um pouco confuso, considerando que a mudança é só dia 15, não?

File: docs/release-notes/custom-io-apps-need-to-add-the-listorders-resource.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ The new `ListOrders` resource was automatically added to all authentications ([r
 
 However, to work as before, in addition to the previous `OMSViewer` resource, accounts with custom IO apps need to add the `ListOrders` resource on their [manifest.json](https://developers.vtex.com/docs/guides/vtex-io-documentation-manifest) file. That way, they will be able to view orders and make [List Orders](https://developers.vtex.com/docs/api-reference/orders-api#get-/api/oms/pvt/orders) requests.
 
-If you need to make the update, go to the `manifest.json` file of your app project and add the `ListOrders` resource to the [policies](https://developers.vtex.com/docs/guides/vtex-io-documentation-policies) field, as shown in the example below:
+This change will be valid for all accounts starting March 15th, 2023. If you need to make the update, go to the `manifest.json` file of your app project and add the `ListOrders` resource to the [policies](https://developers.vtex.com/docs/guides/vtex-io-documentation-policies) field, as shown in the example below:
```
Comment (jumeyermattosarruda) [inline docs/release-notes/custom-io-apps-need-to-add-the-listorders-resource.md:19]:
```suggestion
This change will be valid for all accounts starting on March 15th, 2023. If you need to make the update, go to the `manifest.json` file of your app project and add the `ListOrders` resource to the [policies](https://developers.vtex.com/docs/guides/vtex-io-documentation-policies) field, as shown in the example below:
```

**Review by jumeyermattosarruda — CHANGES_REQUESTED** [review]

File: docs/release-notes/custom-io-apps-need-to-add-the-listorders-resource.md (hunk 2 above)
Comment (Isabella-Veloso) [reply docs/release-notes/custom-io-apps-need-to-add-the-listorders-resource.md:16]: Faz todo sentido, alterei o texto em vários trechos.

**Review by jumeyermattosarruda — APPROVED** [review]

**Pull request merged**